firmware/shared/controls/tvFactor.h

```diff
@@ -0,0 +1,65 @@
+#pragma once
+#include <cmath>
+#include <vector>
+#include <algorithm>
+#include <tuple>
+#include "../util/mappers/lookup_table.h"
+
+namespace ctrl{
+
+template <typename T>
+T tvEnable(T DI_p_steeringAngle, T DI_b_tvEnable){
```

Comment:
should tvEnable be a boolean?

Reply:
That's a very good point. When I looked at the block originally, I wasn't sure since it seems the intention of if statement block was to specifically pass either 0 or the steeringAngle itself to the open_loop_tv block. I also thought: is it a waste to have this function in the first place? All this function does is see if the enable variable is over 0 or not, to which it sends the steering angle or 0 to the lookup table function. If it's going to be a boolean that just enables the use of another function, perhaps instead you would have just a boolean variable that updates on a certain condition elsewhere in the code.

Reply:
the simulink var name is `DI_b...` and the b is boolean, so let's make this a bool